app/controllers/users.py

```diff
@@ -899,13 +899,13 @@ class InActiveUsersView(Resource):
     computed_results = {}  # Dictionary to cache computed results
     current_date = None  # Variable to track the current date
     
-    @admin_required
     def get(self):
```

Comment:
@jnsereko please return the decorator that protects this endpoints with admin rights